lms/djangoapps/instructor/settings/common.py

```diff
@@ -10,35 +10,85 @@ def plugin_settings(settings):
     settings.ANALYTICS_DASHBOARD_URL = ""
     settings.ANALYTICS_DASHBOARD_NAME = _('Your Platform Insights')
     settings.FEATURES.update({
-        # Enable display of enrollment counts in instructor dash, analytics section
+        # .. toggle_name: FEATURES['DISPLAY_ANALYTICS_ENROLLMENTS']
+        # .. toggle_implementation: DjangoSetting
+        # .. toggle_default: True
+        # .. toggle_description: Enable display of enrollment counts in instructor dashboard and
+        #   analytics section.
+        # .. toggle_use_cases: opt_out
+        # .. toggle_creation_date: 2014-11-12
+        # .. toggle_tickets: https://github.com/edx/edx-platform/pull/5838
         'DISPLAY_ANALYTICS_ENROLLMENTS': True,
 
-        # Display the 'Analytics' tab in the instructor dashboard for CCX courses.
-        # Note: This has no effect unless ANALYTICS_DASHBOARD_URL is already set,
-        #       because without that setting, the tab does not show up for any courses.
+        # .. toggle_name: FEATURES['ENABLE_CCX_ANALYTICS_DASHBOARD_URL']
+        # .. toggle_implementation: DjangoSetting
+        # .. toggle_default: False
+        # .. toggle_description: Display the 'Analytics' tab in the instructor dashboard for CCX courses.
+        #   Note: This has no effect unless ANALYTICS_DASHBOARD_URL is already set, because without that
```

Comment:
Optional update: In place of `Note:`, I've been putting notes like this in `toggle_warnings`.

Reply:
FYI: Marking this optional work as resolved so we can more quickly land this.